index.js

```diff
@@ -1,8 +1,28 @@
-const http = require('http');
-const httpProxy = require('http-proxy');
+const http = require("http");
```

Comment:
Do we need to require `http` here? I don't see it anywhere referenced in the code.